### PR TITLE
Fall back to intro if no alt intro has been entered

### PR DESCRIPTION
**Pull request by helenb**
https://projects.torchbox.com/projects/tbxcom-redesign/tickets/150

### CODE REVIEW

File: src/templates/person/person.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ class PersonPage extends React.Component {
     this.title = `${props.firstName + ' ' + props.lastName ||
       ''} [${props.intro || ''}]`
     this.altTitle = `${props.firstName + ' ' + props.lastName ||
-      ''} [${props.altIntro || ''}]`
+      ''} [${props.altIntro || props.intro}]`
```
Comment (chris-lawton) [inline src/templates/person/person.js]: does this just fail silently if neither are entered?

Comment (helenb) [reply src/templates/person/person.js]: good question - it wasn't supposed to be completely empty as it caused a glitch on hover. Probably needs further thought.

**Comment by helenb** [comment]
@chris-lawton now updated.

**Comment by chris-lawton** [comment]
Looks good!

**Pull request merged**